cmd/experimental/kjobctl/apis/v1alpha1/volumebundle_types.go

```diff
@@ -17,11 +17,37 @@ limitations under the License.
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type MountPoint struct {
+	// Inlines Container’s volume mounts.
+	corev1.VolumeMount `json:",inline"`
+
+	// Environment variables pointing to locations on the mount.
+	// These values will be copied to pods/containers environment variables.
+	// +listType=map
+	// +listMapKey=name
+	// +optional
+	Env []corev1.EnvVar `json:"env,omitempty"`
```

Comment:
`cirev1.EnvVal` might be too much, since it's likely we are only going to use the `name` a set of strings should be sufficient.